fe1-web/src/features/home/screens/Launch.tsx

```diff
@@ -29,17 +26,20 @@ const styles = StyleSheet.create({
   } as ViewStyle,
 });
 
-const Launch = ({ navigation }: IPropTypes) => {
+const Launch: FunctionComponent<IPropTypes> = ({ navigation }) => {
   const [inputLaoName, setInputLaoName] = useState('');
   const [inputAddress, setInputAddress] = useState('ws://127.0.0.1:9000/organizer/client');
 
+  const connectToTestLao = HomeHooks.useConnectToTestLao();
+  const createLao = HomeHooks.useCreateLao();
```

Comment:
this API + variable name doesn't tell me that this is a function to execute a network operation. 
`CreateLao` could be the network message, a component, or anything else.
Do try to make it obvious to someone reading the code.

Reply:
👍 I changed it to `requestCreateLao` which also is the name of the original function. The request prefix seems to be used for all network-related functions:)